src/UserSpaceInstrumentation/OrbitUserSpaceInstrumentation.h

```diff
@@ -7,6 +7,16 @@
 
 #include <cstdint>
 
+// InitializeInstrumentation needs to be called once after this library is injected into the target
+// process. It sets up the communication to OrbitService.
+extern "C" void InitializeInstrumentation();
+
+// We'll spawn six threads when injecting this library. This happens immediatly after the call to
+// InitializeInstrumentation above. OrbitService will detect the threads and call SetOrbitThreads to
+// set the thread ids such that events from these threads can be ignored in EntryPayload.
+extern "C" void SetOrbitThreads(uint32_t t0, uint32_t t1, uint32_t t2, uint32_t t3, uint32_t t4,
```

Comment:
The parameter type should be `pid_t` as this is what's used when calling the function (pid_t is signed).

Reply:
switched to using pid_t